omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/impl/DelegatingCrudResource.java

```diff
@@ -91,7 +107,15 @@ public Object create(SimpleObject propertiesToCreate, RequestContext context) th
 		
 		T delegate = handler.newDelegate();
 		setConvertedProperties(delegate, propertiesToCreate, handler.getCreatableProperties(), true);
-		ValidateUtil.validate(delegate);
+		
+		try {
+			ValidateUtil.validate(delegate);

```

Comment:
save(delegate) doesn't invoke the validator, the original code was correct and this try catch isn't necessary because wrapping of the errors and sending an http error code is already handled by the framework, see the code in BaseRestController.validationExceptionHandler